### ISSUE TITLE
Cannot train the model with multiple GPU

### ISSUE THREAD

**Issue by chenweize1998** [nlpAThits/hyfi#1]
Hi. I follow the steps listed in README, but when calculating the loss at training time, the code throws an error at https://github.com/nlpAThits/hyfi/blob/587f1402ef941e73970ee5755377ae6a8cabbaae/hyfi/runner.py#L108

When there are 4 GPUs, the dimension of `logits` is `225, 8822` while `one_hot_true_types ` is `900, 8822`. It seems that the true-types tensor is not distributed to different GPUs. Could you fix it?
